src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
+        m_name, m_pid, m_uid, recv_value, sent_value);
+}
```

Comment:
A timestamp could be added.  I suppose the object might get bit more complicated: might need to wrap the whole thing in a parent object like so:
```
{
  "timestamp": time (ms),
  "data": the_array_above
}
```
It can be done; it's up to you, but I'm not sure how much value it adds -> I figure people will be redirecting this output to a log file of their own, and polling() with inotify or something, and the moment they see new data to read(), they are able to optionally record a timestamp in whatever fashion they choose.  Similarly, the output could be getting redirected into some graph database, or graphite for example, and the user will still have their own methods of tagging each item with a timestamp.

Perhaps the only really useful part for adding timestamps are for users who are doing simple logging (no scripts / no on-the-fly processing) for the purposes of analysis at a later date/time.  If you want to go ahead, let me know if you want the normal trace (`-t`) output timestamped as well.